src/components/FileUploaderDetails/index.tsx

```diff
@@ -64,7 +64,11 @@ const StyledUploaderHost = styled.div`
 `;
 
 const StyledDetailsHost = styled.div`
-  z-index: 10001;
+  // We override the 'z-index' of the styled component 'FixedContainer' of 'Blocker' (which is called by 'Modal').
+  // This fixes the problem of showing 'UploadProgressDetails' in front of 'parent' modal in IE.
```

Comment:
This explanation is confusing, isn't it supposed to say, this fixes the problem of showing ... behind modal in IE? Also, file UploadProgressDetails is not really a child of the modal. Probably a sibling. 

The fix seems hacky, but that's ok for now, but the explanation is a big confusing.